README.md

```diff
@@ -32,8 +32,15 @@ ProGuard
 
 If you are using ProGuard you might need to add the following options:
 ```
+-dontnote retrofit2.Platform
+-dontwarn retrofit2.Platform$Java8
 -dontwarn okio.**
 -dontwarn javax.annotation.**
+-keepattributes Signature
+-keepattributes Exceptions
+-keepclasseswithmembers class * {
```

Comment:
@JakeWharton see provided link in #2338:
https://stackoverflow.com/questions/35506111/retrofit-2-not-sending-data-when-proguard-is-enabled